File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    list = []
    i = 4
    while i <=n:
        cnt = 0
        for j in range(1,i+1):
            if i % j ==0:
                cnt += 1
        if cnt >2:
            list.append(i)
        i += 1
    return len(list)

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int count = 0;
        for (int i = 2; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                if (i % j == 0) {count += 1;}
                if (count >= 3) {break;}
            }
            if (count > 2) {answer += 1;}
            count = 0;
        }
        return answer;
    }
}